test/k8s/test.sh

```diff
@@ -68,7 +68,7 @@ sleep 5
 tarianctl get constraints
 
 # test register constraint using annotation
-tarianctl get constraints | grep run=nginx2 | grep pwd
+tarianctl get constraints | awk '/^---$/ {if (block ~ /selector:/ && block ~ /allowedprocesses:\n- regex: pwd/) print block "---"; block=""; next} {block = block $0 ORS}'
```

Comment:
What does this do @pratikjagrut?

Reply:
But now because of your question, I noticed, the output from the old CLI get command and the output from the new CLI get command is quite a bit different.

Old CLI get command:
```bash
➜  tarian git:(main) ✗ ./bin/tarianctl --server-address=localhost:31051 get constraints
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  NAMESPACE         CONSTRAINT NAME                 SELECTOR                        ALLOWED PROCESSES                                                            ALLOWED FILES
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  default     nginx                          matchLabels:run=nginx    regex:pause,regex:tarian-pod-agent,regex:nginx
  default     nginx-files                    matchLabels:run=nginx                                                     /usr/share/nginx/html/index.html:38ffd4972ae513a0c79a8be4573403edcd709f0f572105362b08ff50cf6de521
  default     nginx2-84273e122d-edeaaff3f1   matchLabels:run=nginx2                                                    /usr/share/nginx/html/index.html:edeaaff3f1774ad2888673770c6d64097e391bc362d7d6fb34982ddf0efd18cb
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

New CLI get command:
```bash
➜  tarian git:(refactor.tarianctl) ✗ ./bin/tarianctl --server-address=localhost:31051 get constraints
kind: Constraint
namespace: default
name: nginx
selector:
  matchlabels:
  - key: run
    value: nginx
allowedprocesses:
- regex: pause
- regex: tarian-pod-agent
- regex: nginx
allowedfiles: []
---
kind: Constraint
namespace: default
name: nginx-files
selector:
  matchlabels:
  - key: run
    value: nginx
allowedprocesses: []
allowedfiles:
- name: /usr/share/nginx/html/index.html
  sha256sum: 38ffd4972ae513a0c79a8be4573403edcd709f0f572105362b08ff50cf6de521
---
kind: Constraint
namespace: default
name: nginx2-84273e122d-edeaaff3f1
selector:
  matchlabels:
  - key: run
    value: nginx2
allowedprocesses: []
allowedfiles:
- name: /usr/share/nginx/html/index.html
  sha256sum: edeaaff3f1774ad2888673770c6d64097e391bc362d7d6fb34982ddf0efd18cb
```

I'll need to fix this.
If you've any insights on this please let me know.

Reply:
Ah, I got it. I kept the default value to yaml, so the format was different and so I had to use this method. I'll revert it back.